### PR TITLE
ETQ U non logué, je vois le bouton "Mon Compte Pro" sur la home au lieu de "Me Connecter"

### PR DESCRIPTION
**Pull request by Isalafont**
Remove btn "Creer un compte" and replace it with "Se connecter avec MonComptePro"

Version with `form` around all MonComptePro component
Capture d’écran 2022-11-16 à 16 07 24

---
Version with `form` only around `Button` in MonComproPro Component

Capture d’écran 2022-11-17 à 11 24 45

### CODE REVIEW

**Comment by Isalafont** [comment]
[ETQ U non logué, je vois le bouton "Mon Compte Pro" sur la home au lieu de "Me Connecter"](https://trello.com/c/u18wZcQc/1659-etq-u-non-logu%C3%A9-je-vois-le-bouton-mon-compte-pro-sur-la-home-au-lieu-de-me-connecter)

File: frontend/src/components/templates/Login/index.js [hunk 1]
```diff
@@ -74,16 +75,8 @@ export const loginUrl = `${BACK_HOST}/users/auth/api_gouv${hashToQueryParams({
 
 const LoginButtons = ({ isOnNewEnrollmentPage }) => (
```
Comment (rdubigny) [inline frontend/src/components/templates/Login/index.js:76]: I think that you can remove the LoginButtons component and put the form in the MonComptePro component

File: frontend/src/components/atoms/MonComptePro/MonComptePro.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React, { MouseEvent } from 'react';
+
+type Props = {
+  href?: string;
+  onClick?: (event: MouseEvent<HTMLElement>) => void;
+  className?: string;
+};
```
Comment (rdubigny) [inline frontend/src/components/atoms/MonComptePro/MonComptePro.tsx:7]: none of these props are used, you can remove it.

File: frontend/src/components/templates/Login/index.js [hunk 3]
```diff
@@ -74,16 +75,8 @@ export const loginUrl = `${BACK_HOST}/users/auth/api_gouv${hashToQueryParams({
...
   <div className="login-buttons">
```
Comment (rdubigny) [inline frontend/src/components/templates/Login/index.js:77]: If you delete this component, mind the suppression of the css rule binded to `.login-buttons` if any.

**Review by rdubigny — CHANGES_REQUESTED** [review]

File: frontend/src/components/templates/Login/index.js [hunk 4]
```diff
@@ -115,7 +92,9 @@ export const Login = () => {
             targetApi={targetApi}
             isOnNewEnrollmentPage={isOnNewEnrollmentPage}
           />
-          <LoginButtons isOnNewEnrollmentPage={isOnNewEnrollmentPage} />
+          <div className="login-button">
+            <MonComptePro isOnNewEnrollmentPage={isOnNewEnrollmentPage} />
```
Comment (rdubigny) [inline frontend/src/components/templates/Login/index.js:96]: this props is not used, plz remove it.

**Review by rdubigny — CHANGES_REQUESTED** [review]

File: frontend/src/components/atoms/MonComptePro/style.css [hunk 5]
```diff
@@ -0,0 +1,3 @@
+.fr-connect-group a {
+  color: var(--text-action-blue-france);
```
Comment (rdubigny) [inline frontend/src/components/atoms/MonComptePro/style.css:2]: not needed, the link should be grey when visited. We should stick on the DSFR here.

**Review by rdubigny — CHANGES_REQUESTED** [review]

**Review by rdubigny — APPROVED** [review]

**Pull request merged**